--- gimp-help-2.8.0/configure.ac.orig	2012-05-24 10:40:04.000000000 +0200
+++ gimp-help-2.8.0/configure.ac	2012-07-02 19:47:30.143566188 +0200
@@ -57,12 +57,12 @@
 #  QUICKREFERENCE_ALL_LINGUAS *and* (intersecton) ALL_LINGUAS
 
 if test "x$ALL_LINGUAS" = "x"; then
-  ALL_LINGUAS="ca da de el en en_GB es fi fr hr it ja ko lt nl nn pl ru sl sv zh_CN"
+  ALL_LINGUAS="da de el en en_GB es fi fr hr it ja lt nl nn pl ru sl sv zh_CN"
 else
   ALL_LINGUAS=`echo "$ALL_LINGUAS" | $SED -e 's/[,] */ /g'`
 fi
 if test "x$QUICKREFERENCE_ALL_LINGUAS" = "x"; then
-  QUICKREFERENCE_ALL_LINGUAS="ca de el en fi fr it ja ko ru sv zh_CN"
+  QUICKREFERENCE_ALL_LINGUAS="de el en fi fr it ja ru sv zh_CN"
 fi
 
 AC_SUBST(ALL_LINGUAS)
